plugins/output/adapt/export.js

```diff
@@ -76,6 +78,49 @@ function copyFrameworkFiles(results, filesCopied) {
   });
 }
 
+function writeThemeVariables(results, variablesWritten) {
+  self.getCourseJSON(TENANT_ID, COURSE_ID, function(error, data) {
+    if (error) {
+      return variablesWritten(error);
+    }
+    var themeVariables = data['course'][0].hasOwnProperty('themeVariables')
+      ? data['course'][0].themeVariables
+      : false;
+
+    var themeName = data['config'][0].hasOwnProperty('_theme')
+      ? data['config'][0]._theme
+      : false;
+
+    const destinationFolder = path.join(EXPORT_DIR, 'src', 'theme', themeName);
+
+    if (themeVariables) {
+      database.getDatabase(function (err, db) {
+        if (err) {
+          return variablesWritten(err);
+        }
+
+        db.retrieve('themetype', {name: themeName}, {}, function(err, results) {
+          if (err || (results && results.length != 1)) {
+            return variablesWritten(err);
+          }
+
+          var theme = results[0];
+
+          self.writeThemeVariables(COURSE_ID, theme, themeVariables, destinationFolder, function(error) {
+            if (error) {
```

Comment:
Don't need the anonymous function here – can simply pass in the callback `variablesWritten`.